src/version_map.jl

```diff
@@ -11911,9 +11911,9 @@ const STDLIBS_BY_VERSION = [
             UUID[UUID("d6f4376e-aef5-505a-96c1-9c027394607a")],
             UUID[],
         ),
-        UUID("dc6e5ff7-fb65-4e79-a425-ec3bc9c03011") => StdlibInfo(
+        UUID("ac6e5ff7-fb65-4e79-a425-ec3bc9c03011") => StdlibInfo(
             "JuliaSyntaxHighlighting",
-            UUID("dc6e5ff7-fb65-4e79-a425-ec3bc9c03011"),
```

Comment:
Wait what?  We changed the UUID of a stdlib?!

Reply:
Ah, did we do the vendoring thing, where we change the UUID to prevent clashes?

Reply:
Answering my own question: https://github.com/JuliaLang/julia/commit/c172a646a33a7a500f7fbfff3475c4348bf5b376

Reply:
Yeah, sorry I should've commented to explain